pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -290,6 +290,14 @@ func (opt *Options) Run(args []string) error {
 				FailureOutput: &FailureOutput{
 					Output: fmt.Sprintf("%d error level events were detected during this test run:\n\n%s", errorCount, errBuf.String()),
 				},
+				Properties: &JUnitProperties{
```

Comment:
Is this based on any standard?

Reply:
> Is this based on any standard?

At least the "test-infra schema supports it" and "we already use it for `<testsuite>`" (which test-infra does not support) standards, as linked from the commit messages.